aya/src/bpf.rs

```diff
@@ -770,9 +772,13 @@ impl Bpf {
             })
     }
 
-    /// Returns a map with the given name.
+    /// Takes ownership of a map with the given name.
     ///
-    /// WARNING: This transfers ownership of the map to the user.
+    /// This API is intended for cases where the map must be moved into spawned
+    /// task. For example, when using an [`AsyncPerfArray`]. For map interactions
```

Comment:
`AsyncPerfEventArray` - my bad as I think I typo'd on the original comment

Reply:
Arg I can't spell haha